formatter/src/main/java/org/togetherjava/formatter/tokenizer/Token.java

```diff
@@ -0,0 +1,32 @@
+package org.togetherjava.formatter.tokenizer;
+
+import java.util.List;
+
+/**
+ * Class representing a Token with a given content and a type
+ *
+ * @author illuminator3
+ */
+public record Token(String content, TokenType type) {
+    private static final List<TokenType> displayTypes =
+            List.of(TokenType.IDENTIFIER, TokenType.UNKNOWN, TokenType.STRING, TokenType.COMMENT);
+
+    @Override
+    public String toString() {
+        return type().name() + displayMe();
+    }
+
+    /**
+     * Returns a non-empty string if this token has something to display
+     *
+     * @return the displayed value
+     * @author illuminator3
+     */
+    private String displayMe() {
```

Comment:
`formatContentForDisplay`? `formatContent`? Not sure what a good name is, but I am not a huge fan of `displayMe`. If you want to keep that terminology `display` would be better, but `display` in general sounds quite active - it might put sth on the screen. This doesn't happen here. If you are consistent with the use of the term that's fine though.